src/components/generic/InputField/InputField.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import webexComponentClasses from '../../helpers';
+
+/**
+ * Input Field component.
+ *
+ * @param {object} props  Data passed to the component
+ * @param {string} props.className  Custom CSS class to apply
+ * @param {object} props.style  Custom style to apply
+ * @param {string} props.label Input label
+ * @param {string} props.type Input type
+ * @param {string} props.name Input name
+ * @param {string} props.value Input value
+ * @param {string} props.placeholder Input placeholder
+ * @param {Function} props.onChange Action to perform on input change
+ * @param {boolean} props.isDisabled  Flag indicating input disabled
+ * @returns {object} JSX of the component
+ */
+export default function InputField({
+  className,
+  style,
+  label,
+  type,
+  value,
+  name,
+  placeholder,
+  onChange,
+  isDisabled,
+}) {
+  const cssClasses = webexComponentClasses('input-field', className);
+
+  return (
+    <div className={cssClasses} style={style}>
+      {label && <label htmlFor="input-field">{label}</label>}
```

Comment:
I think we can remove this, `<Select>` does not have an optional label and it needs to be linked to the actual control (htmlfor="input-field" implies that the input has the id `input-field` which it doesn't).